src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -193,31 +207,43 @@ object DemuxFastqs {
       |                 read group.  This column may be omitted.
+      |  - Sample_Project: The name of the project associated with the sample.  This column may be omitted.
       |
       |Additionally, the sample barcode should be specified in a column named 'Sample_Barcode'.  The name of the column
       |containing the sample barcode can be changed using the --column-for-sample-barcode option.  If the sample barcode
       |is present across multiple reads (ex. dual-index, or inline in both reads of a pair), then the expected barcode
       |bases from each read should be concatenated and placed in the 'Sample_Barcode' column.  The concatenation should
       |be in the same order as the order of the reads' FASTQs and read structures given to this tool.
       |
-      |The read structures will be used to extract the observed sample barcode, template bases, and molecular identifiers
-      |from each read.  The observed sample barcode will be matched to the sample barcodes extracted from the bases in
-      |the sample sheet and associated read structures.
+      |## Metadata CSV
+      |In lieu of a sample sheet, a simple CSV file may be provided with the necessary metadata.  This file should
+      |contain the following columns:
+      |  - Sample_ID:   The sample identifier unique to the sample in the sample sheet.
+      |  - Sample_Name: The sample name.
+      |  - Sample_Barcode: The sample barcodes concatenated across all reads.
+      |  - Library_ID:  The library Identifier.  The combination sample name and library identifier should be unique
+      |                 across the samples in the sample sheet.
+      |  - Description: The description of the sample, which will be placed in the description field in the output BAM's
+      |                 read group.  This column may be omitted.
+      |  - Sample_Project: The name of the project associated with the sample.  This column may be omitted.
+      |
+      |## Example Command Line
       |
       |As an example, if the sequencing run was 2x100bp (paired end) with two 8bp index reads both reading a sample
       |barcode, as well as an in-line 8bp sample barcode in read one, the command line would be
-      |  --fastq r1.fq i1.fq i2.fq r2.fq --read-structures 8B92T 8B 8B 100T \
+      |  --inputs r1.fq i1.fq i2.fq r2.fq --read-structures 8B92T 8B 8B 100T \
       |    --sample-sheet SampleSheet.csv --metrics metrics.txt --output output_folder
     """,
   group=ClpGroups.Fastq
 )
 class DemuxFastqs
 (@arg(flag="i", doc="One or more input fastq files each corresponding to a sub-read (ex. index read, read one, read two, fragment).") val inputs: Seq[PathToFastq],
  @arg(flag="o", doc="The output directory in which to place sample BAMs.") val output: DirPath,
- @arg(flag="s", doc="The sample sheet.") val sampleSheet: FilePath,
+ @arg(flag="s", doc="The sample sheet if the metadata CSV file is not given.") val sampleSheet: Option[FilePath] = None,
```

Comment:
I thought of parsing it both ways but I want to inform the user if either file is malformed along with a useful error message (for the sample sheet at least).